Template/Trigger/ElementVisibilityTrigger.web.js

```diff
@@ -310,10 +294,86 @@
             return false;
         }
 
+        function setMutationObserver(triggerEvent) {
+            return function () {
+                if (observeDomChanges && isMutationObserverSupported) {
+                    var targetNode = documentAlias.body;
+                    var config = {attributes: true, childList: true, subtree: true};
+                    observerMutation = new MutationObserver(function (mutationsList) {
+                        mutationObserverCallback(mutationsList, triggerEvent);
+                    });
+                    observerMutation.observe(targetNode, config);
+                }
+            };
+        }
+
+        function mutationObserverCallback(mutationsList, triggerEvent) {
+            for (var index in mutationsList) {
+                var mutation = mutationsList[index];
+                if (mutation.type === 'childList' || mutation.type === 'attributes') {
```

Comment:
just curious to we still need this check as we're only listening to `{attributes: true, childList: true, subtree: true};`?

Reply:
Yes its not needed, removed